src/shared/components/VideoPreview/VideoPreview.styles.tsx

```diff
@@ -56,7 +57,29 @@ export const CoverIcon = styled(Icon)`
   color: ${colors.white};
 `
 
+export const RemoveButton = styled(Button)`
+  position: absolute;
+  top: ${sizes(2)};
+  right: ${sizes(2)};
+  width: 39px;
+  height: 40px;
+  background-color: ${colors.blue};
+  color: ${colors.white};
+  transition: all ${transitions.timings.regular} ${transitions.easing};
+  cursor: pointer;
+  ${CoverIcon} {
+    width: 20px;
```

Comment:
Gotta be careful when setting explicit size for `svg` elements. If you look at the inspector, the `svg` itself has the correct size, but the `span` that contains it is actually `20x22`, it grows additional 2px to the bottom. This actually causes the icon to be vertically misaligned (by 1px but still :P). That's because by default `svg` will have `display: inline;` which makes it's size calculation not straight forward so it's container will be slightly bigger. You can fix that by adding `display: block;` to these styles